bin/minifollowups/pycbc_page_snglinfo

```diff
@@ -84,24 +88,69 @@ else:
 # make a table for the single detector information ############################
 time = sngl_file.end_time
 utc = lal.GPSToUTC(int(time))[0:6]
-data = [[pycbc.results.dq.get_summary_page_link(args.instrument, utc),
-        str(datetime.datetime(*utc)),
-        '%.3f'  % time,
-        '%5.2f' % sngl_file.snr,
-        '%5.2f' % stat,
-        '%5.2f' % sngl_file.rchisq,
-        '%i'    % sngl_file.get_column('chisq_dof'),
-        '%5.2f' % sngl_file.get_column('coa_phase'),
-        '%5.2f' % sngl_file.mass1,
-        '%5.2f' % sngl_file.mass2,
-        '%5.2f' % sngl_file.mchirp,
-        '%5.2f' % sngl_file.spin1z,
-        '%5.2f' % sngl_file.spin2z,
-        '%5.2f' % sngl_file.template_duration,
-       ]]
+
+try:
+    psd_var = sngl_file.psd_var_val
+except:
+    psd_var = None
+
+try:
+    sg_chisq = sngl_file.sgchisq
+except:
+    sg_chisq = None
+
+# This might involve a lot of lines, but it's clear what goes in the output
+# and in what order.
+headers = []
+data = [[]]
+
+# DQ summary link
+if args.include_summary_page_link:
+    data[0].append(pycbc.results.dq.get_summary_page_link(args.instrument, utc))
+    headers.append("Detector&nbsp;status")
+
+# End times
+data[0].append(str(datetime.datetime(*utc)))
+data[0].append('%.3f' % time)
+headers.append("UTC")
+headers.append("End&nbsp;time")
+
+# SNR and statistic
+data[0].append('%5.2f' % sngl_file.snr)
+data[0].append('%5.2f' % sngl_file.get_column('coa_phase'))
+data[0].append('%5.2f' % stat)
+headers.append("&rho;")
+headers.append("Phase")
+headers.append("Stat")
+
+# Signal-glitch discrimators
+data[0].append('%5.2f' % sngl_file.rchisq)
+data[0].append('%i' % sngl_file.get_column('chisq_dof'))
+headers.append("&chi;<sup>2</sup><sub>r</sub>")
+headers.append("&chi;<sup>2</sup>&nbsp;bins")
+if sg_chisq is not None:
```

Comment:
same comment here. if you're going to try-except, may as well do it here.

Reply:
Changed as suggested.